contrib/guix/README.md

```diff
@@ -75,7 +75,7 @@ crucial differences:
 
 1. Since only Windows and macOS build outputs require codesigning, the `HOSTS`
    environment variable will have a sane default value of `x86_64-w64-mingw32
-   x86_64-apple-darwin18` instead of all the platforms.
+   x86_64-apple-darwin19` instead of all the platforms.
```

Comment:
should bump darwin18->darwin19 in dashbls also, I think (separate PR)

Reply:
Why? The upgrade to `darwin19` is needed because Dash Core is leveraging features that older SDKs don't have. `dashbls` has no such need. We could probably _downgrade_ `dashbls`'s minimum Darwin version to match the capabilities actually used in it.

It doesn't have to match the minver of Dash Core, only not exceed it.